### PR TITLE
[cdc_stream] Merge cdc_stream into asset_stream_manager

### PR DESCRIPTION
**Pull request by ljusten**
Adds start and stop commands to asset_stream_manager.
asset_stream_manager will be renamed to cdc_stream next.

### CODE REVIEW

**Review by wurwunchik — APPROVED** [review]

File: asset_stream_manager/start_command.cc
Comment (wurwunchik) [inline asset_stream_manager/start_command.cc:36]: It did not store my comments :(
So, again. Could you use a named constant here for `2`, please?

Comment (wurwunchik) [reply asset_stream_manager/start_command.cc:36]: (Here and below)

File: asset_stream_manager/stop_command.cc
Comment (wurwunchik) [inline asset_stream_manager/stop_command.cc:34]: The same here.

File: asset_stream_manager/start_command.cc
Comment (wurwunchik) [inline asset_stream_manager/start_command.cc:58]: What would happen if there is a variable (for example, outdated from previous runs) and the user defines a new one. Which will survive?

File: asset_stream_manager/start_command.cc
Comment (ljusten) [reply asset_stream_manager/start_command.cc:36]: Yeah, I guess the diff shows the last PR plus this PR. This should go away once the last PR is merged. Please hold off reviewing this PR for now.

File: asset_stream_manager/start_command.cc
Comment (ljusten) [reply asset_stream_manager/start_command.cc:58]: Not sure what you mean. Commands only run once, so there can't be a "previous" ssh_command_.

File: asset_stream_manager/stop_command.cc
Comment (ljusten) [reply asset_stream_manager/stop_command.cc:34]: See above.

File: asset_stream_manager/start_command.cc
Comment (ljusten) [reply asset_stream_manager/start_command.cc:63]: After talking offline, if there's an environment variable set and a command line flag, the command line flag will win.

**Review by wurwunchik — APPROVED** [review]

**Review by wurwunchik — APPROVED** [review]

**Pull request merged**
